samples/snapshot/pom.xml

```diff
@@ -146,6 +146,8 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <version>3.5.2</version>
         <configuration>
+          <forkCount>10</forkCount>
+          <reuseForks>false</reuseForks>
```

Comment:
Same as for `install-without-bom`: There are no tests in this project at the moment, so I don't think this does anything

Reply:
It is. Currently we are runnings tests three times in all three modules.